accel/tcg/tcg-runtime.c

```diff
@@ -169,26 +169,7 @@ void *HELPER(lookup_tb_ptr)(CPUArchState *env)
     return tb->tc.ptr;
 }
 
-#if defined(CONFIG_MIPS_LOG_INSTR)
-/*
- * Print the instruction to log file.
- */
-void HELPER(log_instruction)(CPUArchState *env, target_ulong pc)
-{
-    if (unlikely(qemu_loglevel_mask(CPU_LOG_INSTR) && qemu_log_in_addr_range(pc))) {
-#if defined(CONFIG_RVFI_DII) && defined(TARGET_RISCV)
-        if (env->rvfi_dii_have_injected_insn) {
-            uint32_t insn = env->rvfi_dii_trace.rvfi_dii_insn;
-            target_disas_buf(stderr, env_cpu(env), &insn, sizeof(insn),
-                             PC_ADDR(env), 1);
-        } else
-#endif
-        {
-            /* Disassemble and print one instruction. */
-            log_target_disas(env_cpu(env), pc, -1);
-        }
-    }
-}
+#if defined(CONFIG_CHERI_LOG_INSTR)
```

Comment:
These should probably also move to the new logging source file?

Reply:
These will probably be dropped when I finish the memory logging part.